tests/e2e/tests/checkout/checkout-block-place-order.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { CheckoutPage } from './checkout.page';
+import {
+	SIMPLE_PHYSICAL_PRODUCT_NAME,
+	FREE_SHIPPING_NAME,
+	FREE_SHIPPING_PRICE,
+} from './constants';
+
+const test = base.extend< { pageObject: CheckoutPage } >( {
+	pageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Checkout block → Place Order → customer', () => {
+	test.use( { storageState: 'playwright/.auth/customer.json' } );
+
+	test( 'Guest user can place order', async ( {
+		pageObject,
+		frontendUtils,
+		page,
+	} ) => {
+		await frontendUtils.emptyCart();
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
+		await frontendUtils.goToCheckout();
+		await expect(
+			await pageObject.selectAndVerifyShippingOption(
+				FREE_SHIPPING_NAME,
+				FREE_SHIPPING_PRICE
+			)
+		).toBe( true );
+		await pageObject.fillInCheckoutWithTestData();
+		await pageObject.placeOrder();
+		await expect(
+			page.getByText( 'Your order has been received.' )
+		).toBeVisible();
+	} );
+} );
+
+test.describe( 'Shopper → Checkout block → Place Order → guest', () => {
+	test.use( { storageState: 'playwright/.auth/guest.json' } );
+
+	test( 'Logged in user can place an order', async ( {
```

Comment:
Think the name should be a logged out user.

I think if we remove the above test, and incorporate it into https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tests/e2e/tests/checkout/checkout-block-shipping.block_theme.side_effects.spec.ts#L74-L81 we could move this test into that same file (and rename the file, as it's no longer focused on shipping).

In my opinion, because these tests are so closely linked it makes sense to combine the files.